superset/views/core.py

```diff
@@ -1176,9 +1176,9 @@ def copy_dash(self, dashboard_id):
         dash.owners = [g.user] if g.user else []
         dash.dashboard_title = data["dashboard_title"]
 
+        old_to_new_sliceids: Dict[int, int] = {}
```

Comment:
not changed in this PR, but maybe `old_to_new_sliceids` -> `old_to_new_slice_ids`? Or or `old_to_new_slc_ids`?

Reply:
fixed.